man/sd_device_enumerator_add_all_parents.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0"?>
+<!DOCTYPE refentries PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+
+<refentry id="sd_device_enumerator_add_all_parents" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>sd_device_enumerator_add_all_parents</title>
```

Comment:
I'd like to use add_match_parent(), as the function name is directly related to the filters introduced by the functions.